src/main/java/com/exadel/discount/model/security/User.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.discount.model.security;
+
+import lombok.Data;
+import lombok.Getter;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Getter
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "email")
+    private String email;
+
+    @Column(name = "password")
+    private String password;
+
+    @Enumerated(value = EnumType.STRING)
+    @Column(name = "role")
+    private Role role;
+}
```

Comment:
UPS...  We already. have "users" table.... Do U have any idea how to fix it?

Reply:
The answer was given in the Skype